helpers/getTestimonials.ts

```diff
@@ -0,0 +1,32 @@
+import client from "../.tina/__generated__/client";
+
+export const GetTestimonialsByCategories = async (
+  categories,
+  defaultCategory

```

Comment:
```suggestion
  categories: string[],
  defaultCategory?: string

```

Reply:
Done - type added and changed defaultCategory a optional param